action.yml

```diff
@@ -60,41 +73,56 @@ runs:
         echo "details:"
         cat details.json
         echo "building new json object.."
-        NEWJSON=$(jq '{ "${{ inputs.step_name }}": { "name": "${{ inputs.step_name }}", "status": "${{ inputs.step_status }}", "statusIcon": "'$icon'", "details": . }}' details.json)
+        NEWJSON=$(jq '{ "${{ inputs.step_name }}": { "name": "${{ inputs.step_name }}", "status": "${{ inputs.step_status }}", "statusIcon": "'$icon'", "details_preview": "${{ inputs.step_details_preview }}", "details": . }}' details.json)
         echo $NEWJSON
         echo "building new state..."
         jq ". + $NEWJSON" state.json > updated_state.json
         cat state.json
         cat updated_state.json
         mv updated_state.json state.json
         echo "## Gruntwork Pipelines Status" > $TMPFILE
+        echo "<table><tr><th>Step</th><th>Status</th><th>Output</th></tr>" >> $TMPFILE
         IFS=$'\n'
         for item in $(cat state.json | jq -c '. | values[]'); do
             echo "single item..."
             echo $item
             name=$(echo "$item" | jq -r '.name')
             statusIcon=$(echo "$item" | jq -r '.statusIcon')
             details=$(echo "$item" | jq -r '.details')
+            details_preview=$(echo "$item" | jq -r '.details_preview')
```

Comment:
NIT: You don't need to `echo` here. You can `jq -r '.details_preview' <<< "$item"` I recommend quoting the whole shell substitution as well. i.e. `details_preview="$(...)"